roles/ovirt-host-remove/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Stops the ovirt-ha-agent service
+  service:
+    name: ovirt-ha-agent
+    state: stopped
+    enabled: no
+  when: host_remove_is_hosted_engine
```

Comment:
Where is the fact coming from? Does this really have to be conditional?

Reply:
Before engine calls this ansible playbook, it sets this fact.
Service probably does not have to be conditional, not sure what it does if ovirt-ha-agent does not exists.
Copy & file needs to be conditional. If it is not hosted engine host then this role will fail, which will create warning in logs that does not have to be there.

Reply:
Why not check the presence of the file in the playbook itself then? I am not a big fan of external facts..

Reply:
or use ignore_error: yes

Reply:
Ok, will do.